OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -667,7 +668,23 @@ private void loadData(@NonNull CourseComponent courseComponent) {
         } else {
             // Remove bulk video download if the course has NO downloadable videos
             UiUtil.removeFragmentByTag(CourseOutlineFragment.this, "bulk_download");
-            errorNotification.showError(isVideoMode ? R.string.no_videos_text : R.string.no_chapter_text, null, -1, null);
+            if (isVideoMode) {
+                errorNotification.showError(R.string.no_videos_text, null, -1, null);
```

Comment:
any recommendation for using this message? cuz we should only use this message when the course doesn't have any video.

Reply:
This message is not appropriate for the section that don't have videos, better to remove check and show the same error messages in all cases.

```java
boolean isSpecialExamInfo = courseComponent.getSpecialExamInfo() != null;
                Map<String, String> values = new HashMap<>();
                values.put(Analytics.Keys.SUBSECTION_ID, courseComponent.getBlockId());
                environment.getAnalyticsRegistry().trackScreenView(isSpecialExamInfo ? Analytics.Screens.SPECIAL_EXAM_BLOCK : Analytics.Screens.EMPTY_SUBSECTION_OUTLINE,
                        courseComponent.getCourseId(), null, values);

                errorNotification.showError(R.string.assessment_not_available, null, R.string.assessment_view_on_web, new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                        environment.getAnalyticsRegistry().trackSubsectionViewOnWebTapped(courseComponent.getCourseId(), courseComponent.getBlockId(), isSpecialExamInfo);
                        BrowserUtil.open(getActivity(), courseComponent.getWebUrl(), false);
                    }
                });
```